index.html

```diff
@@ -103,6 +105,40 @@ <h2> Hello World! 😊 </h2>
             <div class="question-mark"></div>
         </div>
 
+        <div class="card jonnie">
+            <div class="canva">
+                <div class="fish">
+                    <div class="body">
+                        <div class="eye"></div>
+                        <div class="fin-1"></div>
+                        <div class="fin-2"></div>
+                        <div class="fin-3"></div>
+                        <div class="fin-4"></div>
+                        <div class="tail"></div>
+                    </div>
+                    <div class="bubble">
+                        <div class="b-1"></div>
+                        <div class="b-2"></div>
+                    </div>
+                </div>
+            </div>
+            <p class="attribute">'made with ❤️ by Jonnie'</p>
+        </div>
+
+        <div class="card jonnie-2">
+            <div class="canvas">
+                <div class="h">
```

Comment:
well done sir, one suggestion for the future - you might wanna name your classes better 😄

Reply:
^ this goes for one letter class names and a few others that aren't that clear